android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -367,6 +368,23 @@ public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCoun
 			{
 				_firstVisibleItem = firstVisibleItem;
 				_visibleItemCount = visibleItemCount;
+				
+				if (listView.getChildAt(0) != null){
```

Comment:
you can do Child child = listView.getChildAt(0), so you dont have to call it twice